src/Components/WebAssembly/Compression/src/Microsoft.AspNetCore.Components.WebAssembly.Build.BrotliCompression.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Could you explain why this is done as a separate executable, and not an MSBuild task in the regular Build project?

Is it because of some dependency that can't be loaded inside MSBuild?

Reply:
There's no Brotli support in netstandard2.0